createrelease.bat

```diff
@@ -30,6 +30,11 @@ copy "OWML.SampleMods\OWML.EnableDebugMode\bin\Debug\OWML.EnableDebugMode.dll" "
 copy "OWML.SampleMods\OWML.EnableDebugMode\bin\Debug\manifest.json" "Release\Mods\OWML.EnableDebugMode\manifest.json"
 copy "OWML.SampleMods\OWML.EnableDebugMode\bin\Debug\default-config.json" "Release\Mods\OWML.EnableDebugMode\default-config.json"
 
+mkdir "Release\Mods\TAIcheat"
```

Comment:
Why is this being included? I assume this is just left from testing?

Reply:
already explained to alek

Reply:
that's just a temporary measure for showcasing functionality from modder's PoV until similar is done to either of owml mods